desktop/diskStorageManager.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+import os from 'os';
+import disk from 'diskusage';
+import drivelist from 'drivelist';
+import { ipcConsts } from '../app/vars';
+
+const getMountPoint = (drive: any): string => {
+  if (drive.mountpoints && drive.mountpoints.length) {
+    return `${drive.mountpoints[0].path}${os.type() === 'win32' ? '\\' : ''}`;
+  }
+  return '/';
+};
+
+type Drive = {
+  id: string,
+  mountPoint: string,
+  label: string
+};
+
+class DiskStorageManager {
+  static getDriveList = ({ event }: { event: any }) => {
+    drivelist.list((error, drives) => {
```

Comment:
the docs of the library states that main usage is through async function

```
const drivelist = require('drivelist');

const drives = await drivelist.list();
console.log(drives);

```

Reply:
There's an issue with version 8+ of this package. Version 6 is installed works as coded.

Reply:
The issue is with electron and version 8 of this package.